client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -106,9 +142,9 @@ class CustomContentTypes extends Component {
 
 	renderPortfolioSettings() {
 		const { translate } = this.props;
-		const fieldLabel = translate( 'Portfolios' );
+		const fieldLabel = translate( 'Portfolio projects' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Portfolio Projects' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).